lib/openhab/core/dsl/rule/automation_rule.rb

```diff
@@ -237,6 +237,19 @@ class << event
             event.item = inputs&.dig('triggeringItem')
           end
 
+          #
+          # Patch event to decorate event.item with our item wrapper
+          #
+          # @param [OpenHAB Event] event patch
+          #
+          def decorate_event_item(event)
+            class << event
+              def item
+                EntityLookup.decorate_items(super).first
```

Comment:
Would it make sense to split the `EntityLookup.decorate_items` method into two methods: `::decorate_items` that take an array of items, and `::decorate_item` that takes a single item (and is called by `::decorate_items` for each item)? Then you could call the second method directly here.

Reply:
Good idea!